mmv1/products/blockchain/product.yaml

```
@@ -0,0 +1,25 @@
+# Copyright 2019 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Product
+name: Blockchain Node Engine
```

```suggestion
name: BlockchainNodeEngine
```

Once you've fixed the YAML errors and you're able to generate the downstream locally you might see something unexpected about the new generated files. Check what that unexpected thing is, and then make this change to fix it.